### ISSUE TITLE
Option to exclude files

### ISSUE THREAD

**Issue by harrietrs** [paygoc6/action-pull-request-another-repo#16]
A functionality to ignore some folders and files, such as the hooks directory might be really useful.
I am trying to use this action to copy one whole repo to another repo, but I don't want the hooks to create an infinite loop/ fail when the PR is merged.

**Comment by gorrelljd21** [comment]
This would be incredibly useful - were you able to find a work around?

**Comment by harrietrs** [comment]
No, unfortunately not, I had to make my [own (very janky) action](https://github.com/marketplace/actions/copy-pr-action). It does a bunch of other stuff differently as well, and is extremely vulnerable to failure!
